lib/widgets/exampleLocation.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart';
+// import 'package:location/location.dart';
```

Comment:
Esse import será utilizado no futuro? Se não for, creio que seja melhor retirá-lo.

Reply:
> Além dos prints e imports que não estão sendo utilizados, está muito bom. Uma sugestão que pode ser de discussão junto com outros membros é para padronizar a linguagem utilizada nos prints de status.
> 
> Na função `_enable` você utiliza os prints em inglês, mas na função `_permission` você utiliza os prints em português. Acho que padronizar a linguagem desses prints pode tornar o código menos ambíguo no futuro.
> 
> Um último comentário, nessa branch você também está utilizando as imagens `images_solid.png` e `user_figure.png`? Se elas não estiverem sendo utilizadas creio que seja melhor apagá-las.
> 
> De resto eu gostei bastante da implementação. Ficou fácil a compreensão do código.

No geral, esses foram os erros bobos que cometi hahaha, irei corrigi-los por aqui!

Reply:
Aliás, bem observado!

> Concordo com o que o @Franreno disse, acho que devemos padronizar os **print**s. Na minha opinião eles poderiam ser todos em inglês.
> 
> Outra coisa que podemos padronizar é colocar uma pasta para cada serviço dentro do diretório [lib/utils/services](https://github.com/zenitheesc/Zenith-Monitor/tree/utils/location/lib/utils/services), então ao invés de termos **lib/utils/services/location.dart** faríamos, por exemplo, **lib/utils/services/location/location.dart**. Eu o @Franreno vamos criar algumas **exceptions** para os nossos serviços e isso será colocado em arquivos separados dentro dessas pastas. Dessa forma, teríamos algo assim
> 
> ├── services
> │   ├── authentication
> │   │   ├── authentication.dart
> │   │   └── authentication_exceptions.dart
> │   └── location
> │       └── location.dart

Fechou, irei organizar assim também então!